lib/msal-core/src/utils/CryptoUtils.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-
+import { Base64 } from "js-base64";
```

Comment:
As noted in the original Github issue, we previously removed `js-base64` due to compatibility issues, so we may need to come up with another solution for this.